jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/AbstractWebAppMojo.java

```diff
@@ -439,6 +440,7 @@ protected void startJetty()
                 break;
             }
             case HOME:
+            case DISTRO:
```

Comment:
Please produce a warning log message if using DISTRO, encouraging use of HOME instead.

Reply:
This is the bit I'm not in agreement with. The mode was chosen as "DISTRO" not because the distribution was called "jetty-distribution" but because it's the _distribution_. If the name of the distribution changes to "jetty-home" or "jetty-install" or just "jetty", it's still the distribution and shouldn't require the maven plugin to be constantly updated.

Reply:
I would be more in favour of changing it to be "EXTERNAL" than I would be changing it to "HOME".